src/Menu.tsx

```diff
@@ -115,12 +149,16 @@ export const MenuItem: FunctionComponent<IMenuItem> = props => {
       css: styles,
     },
     [
-      icon && <Icon key="menu-icon" icon={icon} pr={title || subtitle ? 10 : 0} />,
+      icon && (
+        <Flex key="menu-icon" alignItems="center" justifyContent="center" width="20px" pr={title || subtitle ? 10 : 0}>
+          <Icon icon={icon} />
+        </Flex>
+      ),
       (title || subtitle) && (
-        <span key="menu-title">
-          {title && <Box>{title}</Box>}
-          {subtitle && <Box>{subtitle}</Box>}
-        </span>
+        <Box as="span" key="menu-title">
```

Comment:
This raises an interesting question. So here it is fine to use a normal `span` because we don't need/want any extra styling. But should we start avoiding using native html elements in favor of things like Box? I say we should only use Box when necessary, so in situations like this to leave it as `span`. Another example is if we want a normal unstyled `div`, we should just use `div` and not have to use `Box`

Reply:
That is a really good question! I think for the structure elements (blocks) that have almost no styling we should use `div` for blocks and `span` fro inline stuff. But for the elements like an alert box, modal window etc. `Box` would be preferrable.